deploy/helm/sumologic/conf/metrics/metrics.conf

```diff
@@ -10,6 +10,15 @@
   @label @DATAPOINT
 </match>
 <label @DATAPOINT>
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (eq .Capabilities.KubeVersion.Minor "16") }}
```

Comment:
what about 1.17 and above? Or these metrics only apply for 1.16?

Reply:
I think they are getting removed in 1.17
> Beta release target 1.17
All previously marked deprecated metrics will be removed from the codebase.

in https://github.com/kubernetes/enhancements/issues/1206

Reply:
They are getting removed, but i believe both metrics co-exist in 1.14, 1.15 and 1.16

Reply:
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#deprecated-metrics -  you can see that they deprecated the old and added the new in 1.14.  So, i think this version check should be for 1.14 and above.

Reply:
I just realized that too, let me update the condition. Thanks!

Reply:
hmm, what should we do for the non-helm yaml file? it looks like CI added the filter to the generated file by helm template